### PR TITLE
add 'created_at' to HelixUser

### PR DESCRIPTION
**Pull request by Rydann**
Type: Bugfix
Fixes: #211

Add the missing `created_at` response field that was added with the Developer Day in November 2020.

Ref: https://twitch.uservoice.com/forums/310213-developers/suggestions/37682716-add-created-timestamp-to-helix-users

### CODE REVIEW

File: packages/twitch/src/API/Helix/User/HelixUser.ts [hunk 1]
```diff
@@ -124,6 +125,13 @@ export class HelixUser implements UserIdResolvableType, UserNameResolveableType
 		return this._data.view_count;
 	}
 
+	/**
+	 * The date when the user was created, i.e. when they registered on Twitch.
+	 */
+	get creationDate(): string {
+		return this._data.created_at;
```
Comment (d-fischer) [inline packages/twitch/src/API/Helix/User/HelixUser.ts:132]: All timestamps in this library are wrapped in a `Date` object. Please change this getter accordingly.

**Review by d-fischer — CHANGES_REQUESTED** [review]
In addition to the given change, please rebase this onto `version/4.4`.

Also, the issue you created should really be of the feature request type. If you had used the feature request template, you would have known that you should be waiting for a response between issue and pull request creation (and you wouldn't have to rebase now).

**Comment by Rydann** [comment]
Yeah sorry my bad, fixed.

**Comment by d-fischer** [comment]
Merging is not rebasing. This PR now pollutes the commit history unnecessarily, especially for such a small change.

**Comment by Rydann** [comment]
Will submit a new and proper pull request, remind me to never try to submit one again after being awake for too long. 🙃

**Pull request closed without merging**